pephub/routers/views/base.py

```diff
@@ -51,11 +51,28 @@ async def main(
 
 
 @views.get("/login/success")
-async def sucessful_login(request: Request, code: str):
+async def successful_login(request: Request, code: str):
     return templates.TemplateResponse(
         "login_success.html",
         {
             "request": request,
             "code": code,
         },
     )
+
+
+@views.get("/login/device/success")
+def login_success(request: Request,
+                  session_info: dict = Depends(read_session_cookie),
```

Comment:
I added this, because I am using base template. So after login you will enter page like this: 
![image](https://user-images.githubusercontent.com/41573628/226638980-208f4183-c514-4e10-91b3-638ad82e0f57.png)